src/client/components/HyfBigLogo/HyfBigLogo.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './HyfBigLogo.style.css';
+import hyfLogo from './hyfLogo.png';
+import { PropTypes } from 'prop-types';
+
+export default function HyfBigLogo(props) {
+  const { title, img } = props;
+  return (
+    <div className="hyf">
+      <div>{img && <img className="hyf-logo" src={hyfLogo} alt="logo" />}</div>
+      <h1 className="app-title">{title}</h1>
+    </div>
+  );
+}
+
+HyfBigLogo.propTypes = {
+  title: PropTypes.string.isRequired,
+  img: PropTypes.node,
+};
+
+HyfBigLogo.defaultProps = {
+  img: 'https://tinyurl.com/yy3pfhp2',
```

Comment:
Do not use arbitrary images as default ones. Defaults should look nicely inside the app and are specific to the application. Ask a designer for default images if they can provide them.

Reply:
Also, this probably doesn't even need to be a prop since this component is basically as static image that won't be changed.

Reply:
"img proptypes removed'